docs/architecture/workflow/Modeler/README.md

```diff
@@ -45,7 +45,7 @@ They are directly mapped from task columns in Task table.
 Diagrams are stored in user local directory, which is defined in kitodo_config.properties:
 
 ```
-diagramsFolder = path/to/diagram/directory
+directory.diagrams = path/to/diagram/directory
 ```
 
-Additionaly there was added Workflow table which has two columns - title and file. Title is id of Process defined in this diagram and file is a file name of this diagram.
\ No newline at end of file
+Additionally there was added Workflow table which has two columns - title and file. Title is id of Process defined in this diagram and file is a file name of this diagram.
```

Comment:
Last line missing

Reply:
Which line?

Reply:
You need a last break in every file. Git hub is showing me, that this file misses it. It was already missing before by the way

Reply:
I don't see anything.

Git breaks lines automatically according to display settings: https://github.com/kitodo/kitodo-production/blob/0b5e9e066801baa91b6e65ce93f7fc3af5262074/docs/architecture/workflow/Modeler/README.md

If you want that I click there enter - tell me on which word it should be.

Reply:
There is just a last empty line needed. Don't you see the warning in your pull request?

Reply:
No, that's why I ask. Normally I see warnings but here is no warning ;) I will add it.

Reply:
Perfekt, now it's fine

Reply:
Perfekt, now it's fine